packages/tests/e2e/auth.spec.ts

```diff
@@ -60,4 +61,92 @@ test.describe("Authentication", () => {
 
     await expect(formPage.successState).toBeVisible();
   });
+
+  test("Reset password flow", async ({ page, context }) => {
```

Comment:
The top level test suite is `Authentication` - this is actually `Recovery`. Can you add a new top level `describe` called `Recovery` and place this scenario there?